styles.css

```diff
@@ -0,0 +1,95 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+body {
+  height: 812px;
+  width: 375px;
+  background-color: #fff;
+}
+
+.nav {
+  display: block;
+  height: 70px;
+  margin: 15px;
+  padding: 0;
+  top: 0;
+  width: 96%;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 10px 15px;
+}
+
+.logo {
+  height: 45px;
+  width: 7%;
+  float: left;
+  color: #36b37e;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.menubar .fa-bars {
+  float: right;
+  font-size: 30px;
+  margin-right: 1.5%;
+  color: #000;
+}
+
+.flexcontainer {
+  height: 80%;
+  display: flex;
+  margin: 65px 2px;
```

Comment:
I think you should remove this margin or make it 0  because this margin you added is too much and created a wide space here 👍